src/component/mxgraph/config/StyleConfigurator.ts

```diff
@@ -179,6 +179,10 @@ export default class StyleConfigurator {
     style[mxConstants.STYLE_HORIZONTAL] = false;
     style[mxConstants.STYLE_SWIMLANE_LINE] = 0; // hide the line between the title region and the content area
 
+    // TODO manage lane text area rendering. there is no Label neither the size available (we have only attribute name="Text of the Label")
+    // perhaps it can be calculated as a difference of starting point (either x or y) between pool, lane, sub-lane ?
+    style[mxConstants.STYLE_STARTSIZE] = 30;
```

Comment:
:question: did you try to decrease this value
`30` is used for `Pool` because this seems to be the value a lot of modelers use.
Here we can do whatever we want for regular lanes
`30` may be needed for lanes that have sub-lanes as in that case, the lane container seems to behave as the pool
:warning: put it in a constant

Reply:
with 30 it is centered vertically - tested with miwg-test-suite Reference files and BPMN editor
constant added, for pool also

Reply:
ok perfect!